components/VC/MosipVCItem/VcItemController.tsx

```diff
@@ -50,19 +47,13 @@ export function useVcItemController(
   const storeErrorTranslationPath = 'errors.savingFailed';
   let generatedOn = useSelector(service, selectGeneratedOn);
   if (props.vcMetadata.isFromOpenId4VCI()) {
-    context = useSelector(service, esignetSelectContext);
-    isKebabPopUp = useSelector(service, esignetSelectKebabPopUp);
-    generatedOn = useSelector(service, esignetSelectGeneratedOn);
-    emptyWalletBindingId = useSelector(
-      service,
-      esignetSelectEmptyWalletBindingId,
-    );
+    context = useSelector(service, selectContext);
```

Comment:
This looks like duplicate assignment as the same is done in line:40

Reply:
true didn't notice that. Thanks for pointing it out